test/parallel/parallel.status

```diff
@@ -79,6 +79,8 @@ test-domain-no-error-handler-abort-on-uncaught-8: PASS, FLAKY
 test-domain-no-error-handler-abort-on-uncaught-9: PASS, FLAKY
 test-domain-throw-error-then-throw-from-uncaught-exception-handler: PASS, FLAKY
 test-domain-with-abort-on-uncaught-exception: PASS, FLAKY
+https://github.com/nodejs/node/issues/56726
+parallel/test-buffer-tostring-range: PASS
```

Comment:
```suggestion
parallel/test-buffer-tostring-range: SKIP
```

Reply:
why pass over skip?

Reply:
Oh you are right, I though I put skip lol